src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const employees = document.querySelectorAll('li');
```

Comment:
You don't need this line of code.
To extract all `li` from your `list` object write `list.children`. (ignores text and comments)